src/app/models/form.server.model.ts

```diff
@@ -145,7 +145,7 @@ const compileFormModel = (db: Mongoose): IFormModel => {
         minlength: [4, 'Form name must be at least 4 characters'],
         maxlength: [200, 'Form name can have a maximum of 200 characters'],
         match: [
-          /^[a-zA-Z0-9_\-./() ]*$/,
+          /^[a-zA-Z0-9_\-./() &`;'"]*$/,
           'Form name cannot contain special characters',
```

Comment:
Might want to change the error message, but I know this won't be seen anywhere haha so 🤷